[BUG] Not all service benchmarks pass as expected

# Bug Description

Not all service benchmarks pass as expected

## **Output of `OLS` version**

0.2.1

## **Describe the bug**

Benchmarks failures:

```
======================================================= short test summary info ========================================================
FAILED tests/benchmarks/test_docs_summarizer.py::test_summarize_empty_history - AttributeError: 'DocsSummarizer' object has no attribute 'summarize'
FAILED tests/benchmarks/test_docs_summarizer.py::test_summarize_no_history - AttributeError: 'DocsSummarizer' object has no attribute 'summarize'
FAILED tests/benchmarks/test_docs_summarizer.py::test_summarize_history_provided - AttributeError: 'DocsSummarizer' object has no attribute 'summarize'
FAILED tests/benchmarks/test_docs_summarizer.py::test_summarize_history_truncation - AttributeError: 'DocsSummarizer' object has no attribute 'summarize'
FAILED tests/benchmarks/test_docs_summarizer.py::test_summarize_no_reference_content - AttributeError: 'DocsSummarizer' object has no attribute 'summarize'
FAILED tests/benchmarks/test_question_validator.py::test_validate_question_short_question - AssertionError: ('p1', 'm1', {'max_tokens_for_response': 4})
FAILED tests/benchmarks/test_question_validator.py::test_validate_question_medium_question - AssertionError: ('p1', 'm1', {'max_tokens_for_response': 4})
FAILED tests/benchmarks/test_question_validator.py::test_validate_question_long_question - AssertionError: ('p1', 'm1', {'max_tokens_for_response': 4})
FAILED tests/benchmarks/test_token_handler.py::test_limit_conversation_history_short_history - TypeError: argument of type 'int' is not iterable
FAILED tests/benchmarks/test_token_handler.py::test_limit_conversation_history_long_history - TypeError: argument of type 'int' is not iterable
FAILED tests/benchmarks/test_token_handler.py::test_limit_conversation_history_huge_history - TypeError: argument of type 'int' is not iterable
========================================= 11 failed, 74 passed, 5 warnings in 61.06s (0:01:01) =========================================
make: *** [Makefile:69: benchmarks] Error 1

```

## **To Reproduce**

Steps to reproduce the behavior:

1. run `pdm run benchmarks`

--- a/tests/benchmarks/test_question_validator.py
+++ b/tests/benchmarks/test_question_validator.py
@@ -24,7 +24,12 @@ def perform_question_validation_benchmark(benchmark, question):
     # be performed
     llm_loader = mock_llm_loader(
         None,
-        expected_params=("p1", "m1", {GenericLLMParameters.MAX_TOKENS_FOR_RESPONSE: 4}),
+        expected_params=(
+            "p1",
+            "m1",
+            {GenericLLMParameters.MAX_TOKENS_FOR_RESPONSE: 4},
+            False,
+        ),
     )
 
     # check that LLM loader was called with expected parameters

--- a/tests/mock_classes/mock_llm_loader.py
+++ b/tests/mock_classes/mock_llm_loader.py
@@ -27,7 +27,7 @@ def mock_llm_loader(llm=None, expected_params=None):
         # if expected params are provided, check if (mocked) LLM loader
         # was called with expected parameters
         if expected_params is not None:
-            assert expected_params == args, expected_params
+            assert expected_params == args, (expected_params, args)
         return MockLLMLoader(llm)
 
     return loader

